tests/e2e/suite_test.go

```diff
@@ -150,7 +150,7 @@ var _ = SynchronizedAfterSuite(func() {
 // of output are not legal JSON
 func saveLogs(buf *bytes.Buffer, logsType, specName string, output io.Writer, capLines int) {
 	scanner := bufio.NewScanner(buf)
-	scanner.Buffer(make([]byte, 0, 64*1024), 1024*1024)
+	scanner.Buffer(make([]byte, 0, 4096), 1024*1024)
```

Comment:
Here we're reducing the size not increasing :S

Reply:
>  It also ensures the starting buffer is 4k to avoid wasting memory.